flink-runtime/src/test/java/org/apache/flink/runtime/leaderelection/TestingLeaderRetrievalService.java

```diff
@@ -61,8 +61,6 @@ public void stop() throws Exception {
 	public void notifyListener(String address, UUID leaderSessionID) {
 		if (listener != null) {
 			listener.notifyLeaderAddress(address, leaderSessionID);
-		} else {
-			throw new IllegalStateException("The retrieval service has not been started properly.");
```

Comment:
the commit name doesn't seem to properly reflect the change made; it should be "don't fail if ... service has _not_ been started"